global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview5-011568",
+    "dotnet": "3.0.100-preview6-012264",
```

Comment:
@aspnet/build Nate had mentioned to try updating our version of .NET to properly consume Roslyn's `Nullable` rename however when I do that it complains about every version known to man not being available. See the associated Azure devops checks for example failures. Do we plan to update our SDK to preview6 any time soon? If not, how can this be worked around?